compiler/tools/fpp-locate-defs/test/defs/defs-2.fpp

```diff
@@ -12,12 +12,16 @@ module M {
 
   port P
 
+  state machine LedSm
+
   passive component C {
     type T
     array A = [3] U32
     constant a = 0
     enum E { X, Y, Z }
     struct S { x: U32 }
+    state machine LedSm
```

Comment:
For stylistic consistency can we just say SM? The Led part is not necessary.

Reply:
Fixed as specified in previous comment